src/block/__tests__/block.spec.js

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// @flow
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Block from '../block';
+
+describe('Block', () => {
+  it('renders correctly', () => {
+    expect(mount(<Block color="primary200">test</Block>)).toMatchSnapshot();
```

Comment:
So technically we do have some ways to do style snapshots, `styled` is mocked in tests:
https://github.com/uber-web/baseui/blob/master/src/styles/__mocks__/styled.js

and we have a `toHaveStyleRule` custom just matcher that leverages this:
https://github.com/uber-web/baseui/blob/master/src/test/expect-to-have-style-rule.js

that said, it's meant to be called directly on a styled component, whereas what you have here is a wrapper around a styled component. You might have to assert styles on the first child (StyledBlock), or try using enzyme shallow with dive().

Reply:
Thanks for the references. I was able to use `getStyle()` to reach 100% coverage

Should be good now.